src/featureFlags.js

```diff
@@ -18,13 +19,13 @@ export class FeatureFlags extends events.EventEmitter {
 
     this.userId = userId;
     this.network = network;
```

Comment:
Can you remove this and the class parameter?

Reply:
Yes we can, removed in ce99526c48f8be781d5cc560a6b43271755bce68